contrib/packs/actions/setup_virtualenv.yaml

```diff
@@ -19,6 +19,11 @@
       description: "Use Python 3 binary when creating a virtualenv for this pack."
       required: false
       default: false
+    no_download:
+      type: "boolean"
+      description: "Use version of pip which is already on the system instead of downloading the latest version from PyPi when creating a virtual environment."
+      required: false
+      default: false
```

Comment:
Are you sure it should be `false`?

Doesn't it mean we'll create a virtualenv with _upgraded_ & _latest_ pip on `st2 pack install`?

Reply:
Yeah, sorry, should indeed default to true - good catch :)